crates/red_knot_project/src/lib.rs

```diff
@@ -106,6 +118,14 @@ impl Project {
         self.settings(db).to_rules()
     }
 
+    /// Returns `true` if `path` is included in the project and one of the paths that should be checked.
```

Comment:
If "included" means "one of the paths that should be checked", then I find this sentence confusing, both because "included in the project" sounds like it has a different meaning (under the project root?), and the "AND one of the paths that should be checked" also suggests that isn't already part of the meaning of "included". If we are going to use "included" to mean "paths requested for checking", let's define what that means in one place (I would do it above on the struct field) and then be consistent and clear about that.
```suggestion
    /// Returns `true` if `path` is both part of the project, and included (see `included_paths_list`).
```